File: TourNW/app/src/main/java/com/kisubs/tournw/PublicArt.java
package com.kisubs.tournw;


import com.google.android.gms.maps.model.LatLng;

/**
 * Created by kisubs on 2017-02-25.
 */

//website,kml_name,address2,Address,city,prov,pcode,phone,Name,Descriptn,id,category,company,fax,email,social_networks,summary,catname,X,Y
public class PublicArt{
    String name, address, description;
    int id;
    LatLng latlng;

    public PublicArt(String address, String name,  String description, int id, LatLng latlng){
        this.address = address;
        this.name = name;
        this.description = description;
        this.id = id;
        this.latlng = latlng;
    }

    public String getAddress(){
        return address;
    }

    public String getName(){
        return name;
    }

    public String getDescription(){
        return description;
    }

    public int getId(){
        return id;
    }

    public LatLng getLatlng(){
        return latlng;
    }
}

File: TouringNW_ios/TouringNW/PublicArt.swift
//
//  PublicArt.swift
//  TouringNW
//
//  Created by Mike Yoon on 2017-02-25.
//  Copyright © 2017 TouringNW. All rights reserved.
//

import Foundation

class PublicArt{
    var name: String!
    var address: String!
    var description: String!
    var id: String!
    var latLong:LatLong = LatLong();
    var rating: Int!
    
    func getName() -> String{
        return name;
    }
    
    func getAddress() -> String{
        return address;
    }
    func getDescription() -> String{
        return description;
    }
    func getId() -> String{
        return id;
    }
    func getLatLong() -> LatLong{
        return latLong;
    }
    func getRating() -> Int{
        return rating;
    }
}
